packages/zipkin-instrumentation-cujojs-rest/src/restInterceptor.js

```diff
@@ -28,6 +29,8 @@ function request(req, {tracer, serviceName = 'unknown', remoteServiceName}) {
     tracer.recordRpc(method.toUpperCase());
     tracer.recordBinary('http.url', reqWithHeaders.path);
     tracer.recordAnnotation(new Annotation.ClientSend());
+    tracer.recordAnnotation(new Annotation.LocalAddr({port: URL.parse(reqWithHeaders.path).port}));
```

Comment:
the localAddr port would be the client side of the socket (not the server side). So better to leave this out as it confuses things.